ios/AirshipFrameworkProxy/Proxies/AirshipPushProxy.swift

```diff
@@ -178,6 +199,58 @@ protocol AirshipPushProtocol: AnyObject {
     var badgeNumber: Int { get set }
     var autobadgeEnabled: Bool { get set }
     var notificationStatus: AirshipNotificationStatus { get async }
+    var quietTime: [AnyHashable: Any]? { get }
+    var quietTimeEnabled: Bool { get set }
+    func setQuietTimeStartHour(
+        _ startHour: Int,
+        startMinute: Int,
+        endHour: Int,
+        endMinute: Int
+    )
+}
+
+extension AirshipPush: AirshipPushProtocol {}
+
+public struct QuietTimeSettings: Codable {
+    let startHour: UInt
+    let startMinute: UInt
+    let endHour: UInt
+    let endMinute: UInt
+
+    public init(startHour: UInt, startMinute: UInt, endHour: UInt, endMinute: UInt) throws {
```

Comment:
probably want to check end time is later than start time

Reply:
it actually does not need to be checked since you can have it start at 10pm and end at 6am or something